### PR TITLE
Cut branch release-2.8 & backplane-2.3 for cluster-proxy-addon.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

Fixes: https://issues.redhat.com/browse/ACM-2752

### CODE REVIEW

File: ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.3.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.18-linux:
```
Comment (skeeey) [inline ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.3.yaml:6]: this is not need

File: ci-operator/config/stolostron/cluster-proxy/stolostron-cluster-proxy-backplane-2.3.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.18-linux:
```
Comment (skeeey) [inline ci-operator/config/stolostron/cluster-proxy/stolostron-cluster-proxy-backplane-2.3.yaml:6]: 1.19?

Comment (xuezhaojun) [reply ci-operator/config/stolostron/cluster-proxy/stolostron-cluster-proxy-backplane-2.3.yaml]: Fixed.

File: ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.3.yaml (hunk 1 above)
Comment (xuezhaojun) [reply ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.3.yaml]: Fixed

**Comment by xuezhaojun** [comment]
@skeeey all test passed, please take another look! Thanks!

**Comment by xuezhaojun** [comment]
/pj-rehearse pull-ci-stolostron-cluster-proxy-addon-backplane-2.3-e2e

**Pull request merged**